ctru-rs/src/services/sslc.rs

```diff
@@ -8,23 +8,12 @@ pub struct SslC(());
 
 impl SslC {
     /// Initialize the service
-    pub fn init() -> crate::Result<Self> {
+    pub fn new() -> crate::Result<Self> {
         unsafe {
             ResultCode(ctru_sys::sslcInit(0))?;
             Ok(SslC(()))
         }
     }
-
-    /// Fill `buf` with `buf.len()` random bytes
-    pub fn generate_random_data(&self, buf: &mut [u8]) -> crate::Result<()> {
```

Comment:
Is this supposed to get removed?

Reply:
This is a duplicate of `Ps::generate_random_data`. The user should use that one or the `rand` crate, which works via `getrandom`.